samples/memfs/memfs.go

```diff
@@ -656,6 +684,16 @@ func (fs *memFS) WriteFile(
 	return err
 }
 
+func (fs *memFS) FlushFile(
+	ctx context.Context,
+	op *fuseops.FlushFileOp) (err error) {
+	if op.OpContext.Pid == 0 {
+		// FlushFileOp should have a valid pid in context.
```

Comment:
Do we make sure that write page cache is not involved here?

Reply:
Thanks for bringing this up! I've disabled the page cache here: https://github.com/xinmeigui-db/fuse/pull/1/files#diff-6402032862b43bb0e095a85c38f691d44fe7063e2cee851b356bf220dc268a8aR98-R101